mobile/status.go

```diff
@@ -231,7 +231,12 @@ func login(accountData, password, configJSON string) error {
 
 	api.RunAsync(func() error {
 		log.Debug("start a node with account", "key-uid", account.KeyUID)
-		err := statusBackend.StartNodeWithAccount(account, password, &conf)
+		err := statusBackend.UpdateNodeConfigFleet(&conf)
+		if err != nil {
+			return err
+		}
```

Comment:
Mobile doesn't use this method `login`. Instead, it uses the `LoginAccount` method. 
The `LoginAccount` method already uses `UpdateNodeConfigFleet` to update the fleet.

Reply:
yep, we have an issue to switch desktop to new endpoints: https://github.com/status-im/status-desktop/issues/12977